src/lib/gadgets/gadgets.ts

```diff
@@ -482,6 +574,42 @@ const Gadgets = {
    * **Note:** This interface does not contain any provable methods.
    */
   Field3,
+  /**
+   * Division modulo 2^32. The operation decomposes a {@link Field} element in the range [0, 2^64] into two 32-bit limbs, `remainder` and `quotient`, using the following equation: `n = quotient * 2^32 + remainder`.
+   *
+   * **Note:** The Gadget expects the input to be in the range [0, 2^64]. If the input exceeds 64 bits, the gadget is invalid and fails to prove correct execution.
```

Comment:
Two comments:
1) `quotient * 2^32 + remainder` can be at most $2^{64} -1$, not $2^{64}$
2) `divMod32` actually acts as a range check on the input, so I'd tweak the wording from "the gadget becomes invalid" to "the gadget fails" for out-of-range inputs.

```suggestion
   * Division modulo 2^32. The operation decomposes a {@link Field} element in the range [0, 2^64) into two 32-bit limbs, `remainder` and `quotient`, using the following equation: `n = quotient * 2^32 + remainder`.
   *
   * **Note:** The gadget acts as a proof that the input is in the range [0, 2^64). If the input exceeds 64 bits, the gadget fails.
```